projects/kubernetes/README.md

```diff
@@ -18,6 +18,10 @@ Boot Kubernetes master OS image using `hyperkit` on macOS: or `qemu` on Linux:
 ```
 ./boot.sh
 ```
+or
```

Comment:
This should give some sort of indication as to what the following alternative does. Something like `or, to automatically initialise the cluster upon boot with no additional options` would be enough.